README.md

```diff
@@ -1,29 +1,48 @@
 # YJDH services
 
-## kesaseteli
+## kesaseteli employer
 
-YJDH-Kesäseteli
+YJDH-Kesäseteli service for employers to fulfill employee applications
 
-### kesaseteli Development with Docker
+### kesaseteli-employer Development with Docker
 
-1. Copy the contents of `.env.kesaseteli.example` to `.env.kesaseteli` and modify it if needed.
+1. Run `yarn` to install necessary packages
 
-2. Run `yarn kesaseteli` or, if you want to rebuild, then `yarn kesaseteli --build`
+2. Copy the contents of `.env.kesaseteli.example` to `.env.kesaseteli` and modify it if needed.
+
+3. Run `yarn employer` or, if you want to rebuild, then `yarn employer --build`
 
 The Frontend is now running at [localhost:3000](https://localhost:3000)
 The backend is now running at [localhost:8000](https://localhost:8000)
 
 3. If services fail to get up, `yarn clean` might help.
```

Comment:
Now the "The Frontend" and "The backend" rows are shown actually on a single row when viewing the README.md through a markdown viewer and not on different rows. The number of the last entry in the list should be updated too.

To fix this they could e.g. be put under the "3." section:
```
3. Run `yarn employer` or, if you want to rebuild, then `yarn employer --build`
   - The Frontend is now running at [localhost:3000](https://localhost:3000)
   - The backend is now running at [localhost:8000](https://localhost:8000)
4. If services fail to get up, `yarn clean` might help.
```

Reply:
fixed